server/endpoints/Review.test.ts

```diff
@@ -0,0 +1,85 @@
+import mongoose from 'mongoose';
+import { MongoMemoryServer } from 'mongodb-memory-server';
+import express from "express";
+
+import axios from 'axios';
+import { configure } from "../endpoints";
+import { Classes, Reviews } from "../dbDefs";
+
+let mongoServer: MongoMemoryServer;
+let serverCloseHandle;
+
+const testingPort = 8000;
```

Comment:
8000 works, but we may want to put it on different ports to avoid any potential conficts with other tests in the future. I generally try to go for >10k.

This is not a big concern though.